src/clr-angular/forms/datepicker/date-container.ts

```diff
@@ -142,6 +175,15 @@ export class ClrDateContainer implements DynamicWrapper, OnDestroy, AfterViewIni
     );
   }
 
+  private _clearEnabled = false;
+  @Input()
```

Comment:
Is there a spec for this that makes the clear button optional? Shouldn't it always appear if there is a value? 

It seems like an unneeded burden for consumers to have to add the input.